src/medida/reporting/collectd_reporter.cc

```diff
@@ -225,10 +225,11 @@ void CollectdReporter::Impl::Process(Histogram& histogram) {
     {kGauge, snapshot.get98thPercentile()},
     {kGauge, snapshot.get99thPercentile()},
     {kGauge, snapshot.get999thPercentile()},
-    // Put 'sum' on the end as it seems clients are assumed to
+    // Put 'sum', 'count' on the end as it seems clients are assumed to
     // be accessing these metrics by position and we do not
     // want to break them.
     {kGauge, histogram.sum()},
+    {kGauge, histogram.count()},
```

Comment:
Looks like CollectdReporter::Impl::Value wants `value` of type `double`, whereas `count` is uint64_t, so I think we need to explicitly cast it with `static_cast` (or implicitly, similarly to how Meter does it on collectd_reporter.cc:200)